corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -374,38 +374,6 @@ def _get_form_ids_for_user(domain, user_id, is_deleted):
             results = fetchall_as_namedtuple(cursor)
             return [result.form_id for result in results]
 
-    @staticmethod
-    def get_all_forms_received_since(received_on_since=None, chunk_size=500):
```

Comment:
They've been moved to special `ReindexAccessor` classes: https://github.com/dimagi/commcare-hq/blob/master/corehq/form_processor/backends/sql/dbaccessors.py#L68